src/helpers/fetch/fetchFromUri.ts

```diff
@@ -44,6 +44,9 @@ export default async (options: FetchFromUriOptions) => {
     };
 
     const response = await axios.request(requestOptions);
+    if (response && response.headers && 'last-modified' in response.headers) {
+      response.data.lastModified = response.headers['last-modified'];
```

Comment:
In general I try to avoid modifying existing objects, but for now I'm okay to accept exceptions to this rule if fixing them takes too much time or makes code less readable. If you run out of ideas, feel free to add `/* tslint:disable:no-object-mutation */` before the violating statement and the should supress TSLint error.